pkg/chaosctl/cmd/root.go

```diff
@@ -42,44 +43,43 @@ Interacting with chaos mesh
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	err := cm.SetupKlog()
-	if err != nil {
-		log.Fatal("failed to setup klog", err)
-	}
 	rootLogger, flushFunc, err := cm.NewStderrLogger()
 	if err != nil {
-		log.Fatal("failed to initialize logger", err)
+		cm.PrettyPrint("failed to initialize logger: ", 0, cm.Red)
+		cm.PrettyPrint(err.Error(), 1, cm.Red)
+		os.Exit(1)
 	}
 	if flushFunc != nil {
 		defer flushFunc()
 	}
 	cm.SetupGlobalLogger(rootLogger.WithName("global-logger"))
 
-	logsCmd, err := NewLogsCmd(rootLogger.WithName("cmd-logs"))
+	logsCmd, err := NewLogsCmd()
 	if err != nil {
-		rootLogger.Error(err, "failed to initialize cmd",
-			"cmd", "logs",
-			"errorVerbose", fmt.Sprintf("%+v", err),
-		)
+		cm.PrettyPrint("failed to initialize cmd: ", 0, cm.Red)
+		cm.PrettyPrint("log command: "+err.Error(), 1, cm.Red)
 		os.Exit(1)
 	}
 	rootCmd.AddCommand(logsCmd)
 
-	debugCommand, err := NewDebugCommand(rootLogger.WithName("cmd-debug"))
+	debugCommand, err := NewDebugCommand(rootLogger.WithName("cmd-debug"), map[string]debug.Debug{
+		networkChaos: debug.NetworkDebug,
+		ioChaos:      debug.IODebug,
+		stressChaos:  debug.StressDebug,
+		httpChaos:    debug.HTTPDebug,
+	})
 	if err != nil {
-		rootLogger.Error(err, "failed to initialize cmd",
-			"cmd", "debug",
-			"errorVerbose", fmt.Sprintf("%+v", err),
-		)
+		cm.PrettyPrint("failed to initialize cmd: ", 0, cm.Red)
+		cm.PrettyPrint("debug command: "+err.Error(), 1, cm.Red)
 		os.Exit(1)
 	}
 
-	physicalMachineCommand, err := NewPhysicalMachineCommand(rootLogger.WithName("cmd-physical-machine"))
+	rootCmd.Flags().StringVarP(&managerNamespace, "manager-namespace", "n", "chaos-testing", "the namespace chaos-controller-manager in")
+	rootCmd.Flags().StringVarP(&managerSvc, "manager-svc", "s", "chaos-mesh-controller-manager", "the service to chaos-controller-manager")
```

Comment:
It should use PersistentFlags here, and `--manager-namespace/-n` would be a conflict with `--namespace/-n` in command `debug` and `--node/-n` in command logs.

I usually install Chaos Mesh in kubernetes namespace `chaos-mesh` not `chaos-testing`, so when I want to declare the namespace, it not works.

Reply:
Fixed, now the shorthand of `manager-namespace` is `-N`.